### ISSUE TITLE
Delta from nominal for image plate parameters.

### ISSUE THREAD

**Issue by akrygier** [HEXRD/hexrdgui#1215]
In the instrument calibration, it would be nice to have the delta from nominal for the refinable image plate parameters. This way, you could evaluate whether a calibration fit is reasonable.

**Comment by joelvbernier** [comment]
Can you clarify @akrygier?  You mean an error plot?

**Comment by joelvbernier** [comment]
Ah -- I think I see.  This should be resolved once the parameter bound constraints are added.  In short, for TARDIS (and PXRDIP), the X & Z translations are sensitive to the cropping (they are nominally in-plane directions), so those could deviate much more than the engineering tolerances would suggest.  We *should*, however, have a very tight constraint on the Y translations (specifically their difference), as this is directly related to the IP separation.  We should come up with reasonable bounds and apply them as box constraints.  This is staged behind getting the 2θ corrections merged in...

**Comment by akrygier** [comment]
![image](https://user-images.githubusercontent.com/59886900/168317021-701aa6bf-417c-414c-9eb8-fcfe2c4f01b0.png)

This is what I made for my fbxrd code. Just an example. Top row is the position in the coord system. Middle is delta from nominal. Bottom are the limits from nominal. 

Checkboxes set refinability state.

Not saying it should necessarily look like that, but I found it very quick to use.

**Comment by akrygier** [comment]
I agree about the different translations. I think something user-controllable rather than under the hood is preferred. Maybe you can make the text red as a warning if it exceeds some hard-coded threshold.

One more idea: you might consider displaying the image axes in mm with origin defined by the translations (in the "Raw" panel).

**Comment by joelvbernier** [comment]
The only problem with "nominal" values is that are specific to instruments for which they exist (which are for sure the minority).  I think hard bounds in the calibration parameters are probably the best general solution -- as they actually constrain the solution.
